source/gui/settingsDialogs.py

```diff
@@ -2659,11 +2668,24 @@ def makeSettings(self, settingsSizer):
 			label=_("&Remove")
 		).Bind(wx.EVT_BUTTON, self.OnRemoveClick)
 
+		if(self._profile.name):
+			bHelper.addButton(
+				parent=self,
+				# Translators: The label for the import entries from default profile dictionary
+				label=_("&import entries from default profile dictionary")
+			).Bind(wx.EVT_BUTTON, self.onImportEntriesClick)
+
 		sHelper.addItem(bHelper)
 
 	def postInit(self):
 		self.dictList.SetFocus()
 
+	def hasEntry(self, pattern):

```

Comment:
I think this could be named more descriptively and docs added. How about `hasPatternEntry`? Initially I expect the `pattern` argument to be regex. Perhaps the args should be `patternColText`

Reply:
You have got a good point here. I originally used pattern because this is the way the column is named on the dialog.